dotnet/targets/Xamarin.Shared.Sdk.DefaultItems.targets

```
@@ -23,19 +23,23 @@
     <None Include="@(ObjcBindingCoreSource)" />
   </ItemGroup>
 
-  <PropertyGroup Condition="'$(_PlatformName)' == 'MacCatalyst'">
-    <EnableDefaultMacCatalystReleaseEntitlements Condition="'$(EnableDefaultMacCatalystReleaseEntitlements)' == ''">True</EnableDefaultMacCatalystReleaseEntitlements>
-    <EnableDefaultMacCatalystDebugEntitlements Condition="'$(EnableDefaultMacCatalystDebugEntitlements)' == ''">True</EnableDefaultMacCatalystDebugEntitlements>
+  <PropertyGroup Condition="'$(EnableDefaultMacCatalystItems)' == 'true'">
+    <EnableDefaultMacCatalystReleaseEntitlements Condition="'$(EnableDefaultMacCatalystReleaseEntitlements)' == ''">true</EnableDefaultMacCatalystReleaseEntitlements>
+    <EnableDefaultMacCatalystDebugEntitlements Condition="'$(EnableDefaultMacCatalystDebugEntitlements)' == ''">true</EnableDefaultMacCatalystDebugEntitlements>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(EnableDefaultMacCatalystDebugEntitlements)' == 'True' and '$(Configuration)' == 'Debug'">
+  <ItemGroup Condition="'$(EnableDefaultMacCatalystDebugEntitlements)' == 'true' and '$(Configuration)' == 'Debug'">
     <CustomEntitlements Include="com.apple.security.get-task-allow" Type="boolean" Value="true" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(EnableDefaultMacCatalystReleaseEntitlements)' == 'True' and '$(Configuration)' == 'Release'">
+  <ItemGroup Condition="'$(EnableDefaultMacCatalystReleaseEntitlements)' == 'true' and '$(Configuration)' == 'Release'">
     <CustomEntitlements Include="com.apple.security.app-sandbox" Type="boolean" Value="true" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(EnableDefaultMacCatalystItems)' == 'true' and '$(Configuration)' == 'Release'">
+    <UseHardenedRuntime>true</UseHardenedRuntime>
```

This makes it overridable:

```suggestion
    <UseHardenedRuntime Condition="'$(UseHardenedRuntime)' == ''">true</UseHardenedRuntime>
```